curve25519-dalek/src/scalar.rs

```diff
@@ -1952,13 +1952,13 @@ pub(crate) mod test {
 
         // ROOT_OF_UNITY^{2^s} mod m == 1
         assert_eq!(
-            Scalar::ROOT_OF_UNITY.pow(&[1u64 << Scalar::S, 0, 0, 0]),
+            Scalar::ROOT_OF_UNITY.pow([1u64 << Scalar::S, 0, 0, 0]),
             Scalar::ONE,
         );
 
         // DELTA^{t} mod m == 1
         assert_eq!(
-            Scalar::DELTA.pow(&[
+            Scalar::DELTA.pow([
```

Comment:
Are these changes a result of a dev dep update?

Reply:
Probably clippy complaining about a needless borrow after merging #562

Reply:
Ah ok